sentry_sdk/integrations/trytond.py

```diff
@@ -0,0 +1,35 @@
+import sentry_sdk
+import sentry_sdk.integrations
+
+from trytond.exceptions import TrytonException  # type: ignore
+from trytond.wsgi import app  # type: ignore
+
+
+# TODO: trytond-worker, trytond-cron and trytond-admin intergations
+
+
+class TrytondWSGIIntegration(sentry_sdk.integrations.Integration):
+    identifier = "trytond_wsgi"
+
+    def __init__(self):  # type: () -> None
+        pass
+
+    @staticmethod
+    def setup_once():  # type: () -> None
+        def error_handler(e):  # type: (Exception) -> None
+            if isinstance(e, TrytonException):
+                return
+            else:
+                sentry_sdk.capture_exception(e)
```

Comment:
Instead of this, could you use `event_from_exception`? The reason is that then you can set a exception `mechanism` of trytond.

Reply:
I still don't understand that mechanism but I trust it will prove useful in the future. I'm about to upload the changes you requested.

Reply:
I realize the API is suboptimal and should be part of `capture_exception`. However right now we make it a policy to set an exception [mechanism](https://docs.sentry.io/development/sdk-dev/event-payloads/exception/#exception-mechanism) for every integration so a user can see why an error was captured.